packages/components/pie-list/test/component/pie-list.spec.ts

```diff
@@ -1,20 +1,51 @@
 import { test, expect } from '@playwright/test';
 import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
-
-const componentSelector = '[data-test-id="pie-list"]';
+import { type ListProps } from '../../src/defs.ts';
 
 test.describe('PieList - Component tests', () => {
     test('should render successfully', async ({ page }) => {
         // Arrange
         const basePage = new BasePage(page, 'list--default');
+        await basePage.load();
+
+        // Act
+        const listComponent = page.locator('pie-list');
+
+        // Assert
+        await expect(listComponent).toBeVisible();
+    });
+
+    test('should render with variant="compact"', async ({ page }) => {
+        // Arrange
+        const listPage = new BasePage(page, 'list--default');
+        const props: ListProps = {
+            variant: 'compact',
+        };
+
+        await listPage.load({ ...props });
+
+        // Act
+        const listComponent = page.locator('pie-list');
+
+        // Assert
+        await expect(listComponent).toBeVisible();
+        await expect(listComponent).toHaveAttribute('variant', 'compact');
+    });
+
+    test('should render with dividers enabled', async ({ page }) => {
+        // Arrange
+        const listPage = new BasePage(page, 'list--default');
+        const props: ListProps = {
+            dividers: true,
+        };
 
-        basePage.load();
-        await page.waitForTimeout(2500);
+        await listPage.load({ ...props });
 
         // Act
-        const list = page.locator(componentSelector);
+        const listComponent = page.locator('pie-list');
 
         // Assert
-        expect(list).toBeVisible();
+        await expect(listComponent).toBeVisible();
+        await expect(listComponent).toHaveAttribute('dividers');
     });
 });
```

Comment:
Currently you have the following test stories:

```
export const Default = createStory<ListProps>(Template, defaultArgs)();

export const Compact = createStory<ListProps>(Template, {
    ...defaultArgs,
    variant: 'compact',
})();

export const WithDividers = createStory<ListProps>(Template, {
    ...defaultArgs,
    dividers: true,
})();

export const CompactWithDividers = createStory<ListProps>(Template, {
    ...defaultArgs,
    variant: 'compact',
    dividers: true,
})();
```

Given you're just using the default story in the test spec, you can either:
- Remove the unused stories
- Refactor the tests to navigate to the other stories, meaning you no longer have to provide the `props` object in the test (as it'll be set by default in the Story).